programs/programs/urgent_needs/co/test_first_step_savings.py

```
@@ -142,53 +142,35 @@ def test_not_eligible_wrong_relationship(self):
-        """Test ineligibility for children born before Jan 1, 2020"""
-        # Add a child born in 2019 (before cutoff)
-        old_child = HouseholdMember.objects.create(
-            screen=self.screen,
-            relationship="child",
-            age=6,
-            birth_year_month=date(2019, 12, 31),  # One day before cutoff
-        )
+    def test_not_eligible_with_children_0_to_5(self):
+        """Test FirstStepSavings is NOT eligible when household has children aged 0-5"""
+        # Add a 1-year-old child (should make NotFirstStepSavings eligible, and FirstStepSavings ineligible)
+        baby = HouseholdMember.objects.create(screen=self.screen, relationship="child", age=1)
```

Ruff (0.14.8)

Local variable `baby` is assigned to but never used

Remove assignment to unused variable `baby`

(F841)